tin/static/css/edit.css

```diff
@@ -0,0 +1,109 @@
+.collapsible {
+  background-color: #777;
+  color: white;
+  cursor: pointer;
+  padding: 18px;
+  max-width: 90%;
+  width: 90%;
+  border: none;
+  border-radius: 10px;
+  text-align: center;
+  outline: none;
+  font-size: 15px;
+  box-shadow: 6px 6px 12px #777,
+              -6px -6px 12px #ffffff;
```

Comment:
![image](https://github.com/tjcsl/tin/assets/110117391/4533ab33-bf36-4b84-8da8-984593ad4ea3)
I have to disagree, tbh.
Naively applying the same box shadow looks weird (see image). However, if we try to combine both the color of the button and the content, it can be hard to see what's actually the button due to the uniformness of gradients.
That being said, if you think it's a blocker, I can try to get it done.

Reply:
Not a blocker - just a curiosity. What I noticed with the current UI is that the box shadow for the button seems to make it appear "floating" (above the page), while for the content it makes it look inset (deeper into the page). I was wondering if this could be a way to solve that. I'll try to experiment with it a bit more and see if I can get something like that to work - if not, your PR looks good to go.